examples/TestDriver/heap.json

```diff
@@ -0,0 +1,4 @@
+{
+    "appId": "2084764307",
+    "enabled": true
```

Comment:
I think this key needs to be a little more descriptive... perhaps `autoInitEnabled` or something?  If I stumbled on this file, I could have a lot of interpretations of what `enabled` means!

Also, I could see us using this file for a lot of things... maybe we want to start allowing configurable prop capture in here like we do for the globals?  Then it wouldn't need to be added to _code_ per se.

@jnatkins Want to make sure I flag this for you - this will be a non-breaking change, but something we should keep current users up to date on and revise docs etc.

Reply:
over in android land, the (as-yet unofficial) names of these variables are: `heapEnvId`, `heapEnabled`, `heapAutoInit`. The `heap` prefixes can go in this world of `heap.json`, and `enabled` is actually "run instrumentation" so isn't relevant here. So `autoInit`?

Reply:
I’d strongly advise having the variables named consistently, if it’s not a ton of work to refactor. It makes work easier for docs, and allows customers to retain their mental model.